spec/support/factory_girl.rb

```diff
@@ -0,0 +1,23 @@
+RSpec.configure do |config|
```

Comment:
Looks like there are a few solutions to this:https://github.com/bbatsov/rubocop/issues/3284

My vote is for adding a rubocop.yml 
"or add an ignore to your rubocop.yml if you don't want to use the Style/FrozenStringLiteralComment. It is new because this is a new feature in Ruby 2.3."

First I will confirm that it works the way I expect with hound.